sys-firmware/libkrunfw/libkrunfw-4.0.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Linux kernel shared object for libkrun"
+HOMEPAGE="https://github.com/containers/libkrunfw"
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~arm64"
+IUSE="asahi"
+
+MY_KVER="6.4.7"
+
+BDEPEND="
+	dev-python/pyelftools
```

Comment:
This needs python version handling, otherwise it can fail like e.g. this:

```
Generating kernel.c from linux-6.4.7/arch/arm64/boot/Image...
Traceback (most recent call last):
  File "/tmp/portage/sys-firmware/libkrunfw-4.0.0/work/libkrunfw-4.0.0/bin2cbundle.py", line 4, in <module>
    from elftools.elf.elffile import ELFFile
ModuleNotFoundError: No module named 'elftools'
make: *** [Makefile:72: kernel.c] Error 1
```

declare PYTHON_COMPAT and inherit python-any-r1, depend on pyelftools with `python_gen_any_dep` and declare a `python_check_deps()` that makes sure a python version is picked that pyelftools is actually built with:

```
python_check_deps() {
    python_has_version "dev-python/pyelftools[${PYTHON_USEDEP}]"
}
```

sys-apps/flatpak is an example that does this.

Reply:
Ah, thanks, I just looked at it and assumed that it was using some binary tool from it rather than importing.

Reply:
thanks for catching this, should be sorted now :)